fuzzylite/CMakeLists.txt

```diff
@@ -74,7 +74,7 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY bin)
 if(NOT MSVC)
 #TODO: Remove -Werror before release.
 #Add Unix compilation flags
-    set(CMAKE_CXX_FLAGS "-pedantic -Wall -Wextra -Werror ${CMAKE_CXX_FLAGS}")
```

Comment:
Quality change

Reply:
Can you elaborate on this?

Reply:
Usually I expect projects to have bugs fixed rather than swept under the rug.

Reply:
Well, as I just mentioned in #77, it is not a bug. It is a case of Clang. 
> Well, I understand the logic to mark this [the method] as a Undefined Behaviour (UB), even though it is not undefined behaviour. The argument in the method is a `float`, which is automatically promoted to a `double` because of the variadic definition, but since `float != double` then it is marked as UB.
> The problem is that I cannot find a way to have a variadic `float` method without raising a warning.

Any thoughts?

Reply:
> even though it is not undefined behaviour.

That is a bold claim, especially in light of _the standard_ saying explicitly it is.

See paragraph 18.10.3 http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2014/n4296.pdf

Reply:
Could you please point me to a solution where you have a `float` variadic method without raising a warning?

Reply:
@janisozaur, I am planning on merging this PR over the weekend. Have you experimented with the example above? Do you have any useful information that could lead to a better solution? Cheers.

Reply:
@jcrada Would you consider backporting not treating warnings as errors to the 4.0 branch, please..

Reply:
Hi @matthiessnj, can you be more explicit on your request? What do you refer to backporting? What 4.0 branch? Thanks.